tests/cucumber/steps/steps.js

```diff
@@ -4714,6 +4714,45 @@ module.exports = function getSteps(options) {
     }
   );
 
+  When('I make a new simulate request.', async function () {
+    this.simulateRequest = new algosdk.modelsv2.SimulateRequest(
+      [],
```

Comment:
I don't think this is right. `SimulateRequest` requires one argument that's an object. Maybe it happens to work because of JS soft typing, but I'd expect this to be `SimulateRequest({ txnGroups: [] })`

Reply:
ok, type script actually screws me up 😢

Reply:
The problem is this file isn't typescript...

Reply:
see [pr comment response](https://github.com/algorand/js-algorand-sdk/pull/768/commits/fe95c4cbd087bffbfe7703092a28e0936a0e2186)